keyboards/lets_split/keymaps/geripgeri/keymap.c

```diff
@@ -0,0 +1,438 @@
+),
+
+/* Gaming
+ * ,-----------------------------------------.               ,-----------------------------------------.
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      |      |      |               |      |      |      |      |      |      |
+ * |------+------+------+------+------+------|               |------+------+------+------+------+------|
+ * |      |      |      |      | RAISE|      |               |      | LOWER|      |      |      |      |
+ * `-----------------------------------------'               `-----------------------------------------'
+ */
+[_GAMING] = LAYOUT_ortho_4x12(
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, RAISE,   _______,       _______, LOWER,   _______, _______, _______, _______
+)};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
+      if (record->event.pressed) {
+	rgb_mode = RGBLIGHT_MODE_BREATHING + 1;
+	autoshift_enable();
+        set_single_persistent_default_layer(_QWERTY);
+      }
+      return false;
+      break;
+    case LOWER:
+      if (record->event.pressed) {
+        layer_on(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_LOWER);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case RAISE:
+      if (record->event.pressed) {
+        layer_on(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      } else {
+        layer_off(_RAISE);
+        update_tri_layer(_LOWER, _RAISE, _ADJUST);
+      }
+      return false;
+      break;
+    case ADJUST:
+      if (record->event.pressed) {
+        layer_on(_ADJUST);
+      } else {
+        layer_off(_ADJUST);
+      }
+      return false;
+      break;
+    case GAMING:
+      if (record->event.pressed) {
+	rgb_mode = RGBLIGHT_MODE_RAINBOW_SWIRL + 5;
+        autoshift_disable();
+	default_layer_set(1UL<<_GAMING);
+      }
+      return false;
+      break;
+    case NUMPAD:
+      if(record->event.pressed){
+	layer_on(_NUMPAD);
+	PORTB &= ~(1<<0);
+      } else {
+	layer_off(_NUMPAD);
+	PORTB |= (1<<0);
+      }
+      return false;
+      break;
```

Comment:
You could just use an `MO(_NUMPAD)` instead, and use `layer_state_set_user` to handle this.